jgrix-nepal.md

```diff
@@ -0,0 +1,11 @@
+Kathmandu, Nepal
```

Comment:
I have not. However looking through previous classes that have done this, someone has and I thought it looked interesting. This way I won't forget.